docs/api.md

```diff
@@ -150,7 +150,8 @@ Puppeteer is a Node library which provides a high-level API to control Chromium
 Puppeteer looks for certain [environment variables](https://en.wikipedia.org/wiki/Environment_variable) to aid its operations:
 
 - `HTTP_PROXY`, `HTTPS_PROXY`, `NO_PROXY` - defines HTTP proxy settings that are used to download and run Chromium.
-- `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD` - do not download bundled Chromium during installation step.
+- `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD` - do not download bundled Chromium during installation step. (`puppeteer_skip_chromium_download` in `.npmrc` as same)
+- `PUPPETEER_DOWNLOADS_FOLDER` - specify an absolute path to hold downloads. (`puppeteer_downloads_folder` in `.npmrc` as same)
```

Comment:
let's put `PUPPETEER_DOWNLOADS_FOLDER` before `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD`. Alphabetic order will help to navigate the section.